oximeter/db/src/client/mod.rs

```diff
@@ -104,19 +220,34 @@ impl Client {
         let client = reqwest::Client::new();
         let url = format!("http://{}", address);
         let schema = Mutex::new(BTreeMap::new());
-        Self { _id: id, log, url, client, schema, request_timeout }
+        Self {
+            _id: id,
+            log,
+            source: ClientSource::Static(ReqwestClient { url, client }),
+            schema,
+            request_timeout,
+        }
     }
 
-    /// Return the url the client is trying to connect to
+    /// Return the url the client is trying to connect to.
+    ///
+    /// For pool-based clients, this returns "dynamic", as the URL may change
+    /// between accesses.
```

Comment:
If there are many Clickhouse databases with different socketaddresses, which URL should we return here?

I don't want to check out one connection from the pool and give an address that might change next time you connect -- since this seems to be used for debugging anyway, I just figured I'd use a placeholder.

Reply:
I think that's right. This method is actually only used in the integration tests right now, though I was hoping to remove it entirely.

Also, this will likely go away in the medium term anyway, once I rip out the HTTP interface to ClickHouse entirely in exchange for the TCP-based client.